test/openshift/e2e/parallel/1-057_validate_notifications/05-verify-email.yaml

```diff
@@ -5,7 +5,7 @@ commands:
     set -e
     sleep 5
     smtp4dev_pod=$(kubectl get pod -l=app=smtp4dev -o NAME -n $NAMESPACE)
-    exit_code=$(kubectl -n $NAMESPACE exec --stdin --tty "${smtp4dev_pod}" -- /bin/bash \
+    exit_code=$(kubectl -n $NAMESPACE exec --stdin "${smtp4dev_pod}" -- /bin/bash \
```

Comment:
I don't recall right now... was `--tty` causing unexpected outputs like `\r\n`?

Reply:
Hi @bluengo 
This is just for eliminating warning 

> Unable to use a TTY - input is not a terminal or the right kind of file

Reply:
/lgtm